src/TrackInfo.cpp

```diff
@@ -204,6 +204,51 @@ unsigned TrackInfo::MinimumTrackHeight()
    return height + kTopMargin + kBottomMargin + 1;
 }
 
+int TrackInfo::MeasureTotalTracksHeight(TrackList& tracks)
+{
+   if (!tracks.empty())
+   {
+       auto& view = TrackView::Get(*tracks.back().get());
+       return view.GetBottom() + kBottomMargin;
```

Comment:
If GetBottom() is corrected with -1  as I suggested, then I think you would add 1 here.

So the result is not now wrong, but GetBottom() is misleading.  I expect it to mean the y coordindate of the bottom, not one past that coordinate.

Reply:
Probably you are right, will check

Reply:
I checked usage of constants (like kBottomMargin, kTopMargin ...), and I think there may be more work to be done to properly calculate layouts (drawing and sizing, calculation is done twice at least), correcting all the errors. Right now it seems that constants used in UI doesn't respect this correction either, and fixing it in one place and not fixing in others seems odd to me.